internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -29,9 +29,9 @@
 {{- if eq .WorkloadType "Load Balanced Web Service"}}
   PortMappings:
     - ContainerPort: !Ref ContainerPort
-    {{- if .ServiceConnect }} # remove when release
-    {{- if eq .HTTPTargetContainer.Container .WorkloadName}}
-      Name: TargetPort
+    {{- if and .ServiceConnect .SCFeatureFlag}} # remove when release
+    {{- if eq .HTTPTargetContainer.Name .WorkloadName}}
+      Name: target
```

Comment:
Is this field safe to update in the future? For example, is it okay to rename it from `target` to `target1`?

Reply:
yeah i think so. it's only gonna be referenced within the stack. and right now we don't make any assumption based on that elsewhere.